### PR TITLE
League Detail View -- Schedule Table

### PR DESCRIPTION
**Pull request by MCurran16**

### CODE REVIEW

File: assets/js/model/schedule.js.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#= require lib/base.js.coffee
+
+do ->
+
+   class Schedule
+      constructor: (data) ->
+         assertArgs(arguments, Object)
+         @id = data.id
+         @weekNumber = ko.observable(data.week_number)
```
Comment (hunterbrowning) [inline assets/js/model/schedule.js.coffee]: Lets ditch all these date related properties and just have like a timestamp or something.

File: assets/js/templates/views/league-details/league-details.jst.jade [hunk 2]
```diff
@@ -48,10 +48,28 @@ script(type="text/html" id="details-list-template")
 		td.league-detail-text.league-note Balls and awesomeness provided by KC Crew
 			span.league-detail-text.league-note Schedules released a week before season start
 
+script(type="text/html" id="schedule-table-template")
+	<!-- ko if: $index() == 0 -->
+	tr.schedule-table__row.schedule-body-header
+		td.schedule-header-text(colspan="5", data-bind="text: $parent.title()")
```
Comment (hunterbrowning) [inline assets/js/templates/views/league-details/league-details.jst.jade]: don't think you need the () on title. Please remove them if you don't.

File: assets/js/templates/views/league-details/league-details.jst.jade [hunk 3]
```diff
@@ -48,10 +48,28 @@ script(type="text/html" id="details-list-template")
...
+			.team-color-indicator(data-bind="style: {backgroundColor: $data.homeTeamColor}")
+			span.team-name-text(data-bind="text: $data.homeTeamName")
+			a.sprite.icon-information
+		td.schedule-text(data-bind="text: $data.homeTeamScore")
+
 .league-details-container
 	.google-map-container
 		.map-cover
-		.hero-map(data-bind="googleMap: gameLocation")
+		//- .hero-map(data-bind="googleMap: gameLocation")
```
Comment (hunterbrowning) [inline assets/js/templates/views/league-details/league-details.jst.jade]: Should this be removed if its commented out?

File: assets/js/vm/views/league-details/league-details.js.coffee [hunk 4]
```diff
@@ -53,16 +58,22 @@ do ->
...
+         @standingsArray = ko.observableArray([])
+
 
          ###-----------------------------------
             Standings Content
          ----------------------------------###
-         @standingsArray = ko.observableArray([])
+         @leagueSchedules = ko.observableArray([])
+
+         @weekDays = ["Monday", "Tuesday", "Wednesday", \
```
Comment (hunterbrowning) [inline assets/js/vm/views/league-details/league-details.js.coffee]: Lets not store these date related values in the VM. Maybe in the utils folder we can create some type of date utility as I suspect we will need these in multiple places.

File: assets/js/templates/views/league-details/league-details.jst.jade (hunk 3 above)
Comment (MCurran16) [inline assets/js/templates/views/league-details/league-details.jst.jade]: I disabled the functionality for the google map. I will enable it at the end of the project when we get a google map account.

File: assets/js/templates/views/league-details/league-details.jst.jade (hunk 2 above)
Comment (MCurran16) [inline assets/js/templates/views/league-details/league-details.jst.jade]: I do not need it. fixed

File: assets/js/templates/views/league-details/league-details.jst.jade (hunk 3 above)
Comment (hunterbrowning) [inline assets/js/templates/views/league-details/league-details.jst.jade]: Thats right. My bad.

File: assets/js/model/schedule.js.coffee (hunk 1 above)
Comment (MCurran16) [inline assets/js/model/schedule.js.coffee]: fixed

File: assets/js/vm/views/league-details/league-details.js.coffee (hunk 4 above)
Comment (MCurran16) [inline assets/js/vm/views/league-details/league-details.js.coffee]: fixed in the date-utility

**Comment by hunterbrowning** [comment]
LGTM

**Pull request merged**